### ISSUE TITLE
E2e test throws error of permission issue to view project "project-name"

### ISSUE THREAD

**Issue by amitkrout** [openshift/odo#1513]
[kind/bug]

## What versions of software are you using?
- Test Env: OpenShift 4.0, prow setup or aws
- Output of `odo version`: master

## How did you run odo exactly?
make configure-installer-tests-cluster
make test-main-e2e

## Actual behavior
Fails

## Expected behavior
Should not throw error of 
```
[1553433935] odo test suite - 6/67 specs Running command: oc project -q
error: you do not have rights to view project "json-test".
```

## Any logs, error output, etc?
```
[1553433935] odo test suite - 6/67 specs Running command: oc project -q
error: you do not have rights to view project "json-test".

------------------------------
Failure [0.256 seconds]
[BeforeSuite] BeforeSuite 
/go/src/github.com/openshift/odo/tests/e2e/e2e_test.go:47

  Expected
      <int>: 1
  to equal
      <int>: 0

  /go/src/github.com/openshift/odo/tests/e2e/e2e.go:42
```

**Comment by amitkrout** [comment]
#1470

**Comment by kadel** [comment]
https://github.com/redhat-developer/odo/pull/1360 might have fix for this

**Comment by amitkrout** [comment]
Fixed via - #1517
